OutOfSchool/OutOfSchool.AuthServer.Tests/Services/CommonMinistryAdminServiceTests.cs

```diff
@@ -1,18 +1,21 @@
 ﻿using AutoMapper;
 using FluentAssertions;
 using Microsoft.AspNetCore.Identity;
-using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Routing;
```

Comment:
(Excessive usings) Please double-check if all usings here are necessary

Reply:
Fixed. Thank you!